### PR TITLE
👷 create devnet-dump for e2e tests in CI

### PR DESCRIPTION
**Pull request by ofux**
depends on https://github.com/onlydustxyz/starknet-accounts/pull/4

### CODE REVIEW

**Review by tekkac — APPROVED** [review]
Lgtm

**Review by Bernardstanislas — APPROVED** [review]

File: .github/workflows/devnet.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    runs-on: ubuntu-latest
+    name: Build devenet dump
+    steps:
+      - name: Checkout smart-contracts
+        uses: actions/checkout@v3
+
+      - name: Checkout starknet-accounts
+        uses: actions/checkout@v3
+        with:
+          repository: ofux/starknet-accounts
```
Comment (AnthonyBuisset) [inline .github/workflows/devnet.yml:21]: To be modified with OnlyDust ?

File: .github/workflows/devnet.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          [ -f $GITHUB_WORKSPACE/scripts/.env.local ] && rm $GITHUB_WORKSPACE/scripts/.env.local
+          cp ./build/deployed_accounts.txt $GITHUB_WORKSPACE/scripts/.env.local
+          cat $GITHUB_WORKSPACE/scripts/.env.local
+
+      - name: Deploy smart contracts
+        run: |
+          [ -f ./build/deployed_contracts_local.txt ] && rm ./build/deployed_contracts_local.txt
+          ./scripts/deploy.sh -a local_admin -p local -y
+
+      - name: Create devnet dump
```
Comment (AnthonyBuisset) [inline .github/workflows/devnet.yml:63]: I think we should wait here as the dump can take time

**Review by AnthonyBuisset — CHANGES_REQUESTED** [review]
It is gonna be so convenient!
Just a couple of comments to check

File: .github/workflows/devnet.yml (hunk 1 above)
Comment (ofux) [reply .github/workflows/devnet.yml:21]: you're right

File: .github/workflows/devnet.yml (hunk 2 above)
Comment (ofux) [reply .github/workflows/devnet.yml:63]: I don't know _why_ exactly, but it actually **does** wait.

**Pull request merged**